presentation/src/main/res/layout/fragment_write_to_us.xml

```diff
@@ -0,0 +1,254 @@
+            style="@style/Text2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/size_1.5x"
+            android:layout_marginLeft="@dimen/size_3x"
+            android:text="@string/e_mail_with_star"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/layout_eText_phone_number"/>
+
+        <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/layout_eText_email_address"
+            android:theme="@style/Layout_eText_theme_overlay_p_60"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/size_6x"
+            android:layout_marginTop="2dp"
+            android:layout_marginLeft="@dimen/size_3x"
+            android:layout_marginRight="@dimen/size_3x"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/email_address">
+
+            <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/eText_email_address"
+                style="@style/EText_email_address_p60"
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/size_6x"
+                android:hint="@string/enter_email_address"/>
+        </com.google.android.material.textfield.TextInputLayout>
+
+        <TextView
+            android:id="@+id/message"
+            style="@style/Text2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/size_1.5x"
+            android:layout_marginLeft="@dimen/size_3x"
+            android:text="@string/message_with_star"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/layout_eText_email_address"/>
+
+        <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/layout_eText_message"
+            android:theme="@style/Layout_eText_theme_overlay_p_60"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/size_10x"
+            android:layout_marginTop="2dp"
+            android:layout_marginLeft="@dimen/size_3x"
+            android:layout_marginRight="@dimen/size_3x"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/message">
+
+            <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/eText_message"
+                style="@style/EText_p60"
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/size_10x"
+                android:hint="@string/enter_your_message_here"/>
```

Comment:
line after this input seems to be lower than in others, could you check that?

Reply:
Yes, because its like in zepplin design. It is okay or no?

Reply:
if it looks like on the design, then it is ok, however it is just strange